Add `o!mLN4` Tournament Wiki page.

RushFTK opened the pull request:
<!-- 
  - Use [x] to complete the items
  - Remove the items unrelated to your work
  - Add any relevant information you consider useful
  - If there are no reviewers for your language, please mention it explicitly
-->

## Self-check

- [ ] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

RushFTK committed:
Initial update for `o!mLN4` wiki page

RushFTK committed:
Update o!mLN4 wiki page before staff list

RushFTK committed:
update for draft of rules

RushFTK committed:
Update mappools and rulebooks for `omLN4`

RushFTK committed:
Fix CI issue for `o!mln4`

RushFTK committed:
Fix inner url link issue for `omln4`

RushFTK committed:
Zipped file to suit for rules for omln4

RushFTK commented:
`Partipicate` and `Result` parts will update in next PR, happened in few days.

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+
+## Prizes
+
+Prize pool are divided to two parts: one is Regular Prize Pool, allocated to tournament winner; the other one is OII Prize Pool, allocated to event winner.
```
A bit wack to read, miscellaneous changes. Using future tense as this action has not happened yet, although using a present tense should be just as fine. Semantics.

```suggestion
The prize pool will be divided in two parts: one is the **Regular Prize Pool**, allocated to the tournament winner; the other one is the **OII Prize Pool**, allocated to the event winner.
```

Also, in my opinion, I believe we should explain what "OII" means. It was explained in the news post but there isn't any mention of it prior to the prize pool section. A user stumbling upon this will have no idea what that means.

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+
+Prize pool are divided to two parts: one is Regular Prize Pool, allocated to tournament winner; the other one is OII Prize Pool, allocated to event winner.
+
+The Regular Prize Pool starts at 450$ USD, while the OII Prize Pool will start at 100$ USD. Community donations will contribute to both Tournament and OII Prize Pools in the ratio 70/30 respectively.
```
Currency formatting.
```suggestion
The Regular Prize Pool starts at $450 USD, while the OII Prize Pool will start at $100 USD. Community donations will contribute to both Tournament and OII Prize Pools in a 70/30 ratio respectively.
```

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+
+So far, There are **450$ USD** on Regular Prize Pool and **100$ USD** on OII Prize Pool.
+
+Regular Prize Pool are spread among the participants as seen below:
```
More accurate word for this context + singular.
```suggestion
The Regular Prize Pool is distributed among the participants as seen below:
```

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+
+## Ordeal in Illusion
+
+![Ordeal in Illusion banner](img/ordeal-in-Illusion.png "The Ordeal in Illusion banner in forum post")
```
```suggestion
![Ordeal In Illusion banner](img/ordeal-in-Illusion.png "The Ordeal In Illusion banner in the forum post")
```

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+| *4-7th place* | 2 Tickets |
+| *Other participates* | 1 Ticket |
+
+At the end of the tournament, Tickets earned by all participants are tabulated and 3 Tickets will be drawn during a livestream. The owners of these Tickets will win a part of the community-donated OII prize pool!
```
Don't understand what "tabulated" is making reference to. As in counting the tickets from each player? If so, the suggestion below should work better.
```suggestion
At the end of the tournament, tickets earned by all participants are tallied and 3 tickets will be drawn during a livestream. The owners of these tickets will win a part of the community-donated OII prize pool!
```

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+
+At the end of the tournament, Tickets earned by all participants are tabulated and 3 Tickets will be drawn during a livestream. The owners of these Tickets will win a part of the community-donated OII prize pool!
+
+Details can be found on **[Information spreadsheet](https://docs.google.com/spreadsheets/d/1Wl3Q41AI0e36sINVWh6qJHtKcCG_fdju2_0CdBm0DGs/edit?usp=sharing)**.
```
```suggestion
Details can be found on the **[information spreadsheet](https://docs.google.com/spreadsheets/d/1Wl3Q41AI0e36sINVWh6qJHtKcCG_fdju2_0CdBm0DGs/edit?usp=sharing)**.
```

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+### General rules
+
+1. This tournament is a **1v1 global open rank double-elimination tournament**.
+2. Players will first go through Qualifiers, then continuing to the double-elimination bracket.
```
```suggestion
2. Players will first go through Qualifiers, then continue to the double-elimination bracket.
```

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+
+1. This tournament is a **1v1 global open rank double-elimination tournament**.
+2. Players will first go through Qualifiers, then continuing to the double-elimination bracket.
+3. Any members of staff **apart from organisers, mappool selectors, and referees** are allowed to play.
```
```suggestion
3. Any staff members **other than organisers, mappool selectors, and referees** are allowed to play.
```

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+
+### Mappool format
+
+The maps in this tournament will only consist of Long Note (LN) oriented maps, and cover 4 sub-categories of LNs which are used to categorise all mappools after this stage. Details can be found below:
```
```suggestion
The maps in this tournament will only consist of long note (LN) oriented maps, and cover 4 sub-categories of LNs which are used to categorise all mappools after this stage. Details can be found below:
```

MChecaH reviewed (commented):
First part of the review, up to `Mappool format`

RushFTK reviewed (commented):
Apply reviews from MChecaH

RushFTK committed:
Update wiki/Tournaments/o!mLN/4/en.md

Co-authored-by: MChecaH <93517213+MChecaH@users.noreply.github.com>

RushFTK committed:
Apply suggestions from code review

Co-authored-by: MChecaH <93517213+MChecaH@users.noreply.github.com>

RushFTK committed:
Update rulebooks & description based on review.

RushFTK committed:
Fix CI issue for omln4

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+
+### Registrations
+
+1. **Any** rank players from **any** region can register this tournament.
```
Very weird phrasing to refer to the tournament as "open rank". Plus grammar.

```suggestion
1. Players with **any rank** from **any region** can register to this tournament.
```

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+### Registrations
+
+1. **Any** rank players from **any** region can register this tournament.
+2. You must fill [Player Signup form](https://docs.google.com/forms/u/0/d/e/1FAIpQLSfQGnO66J7b22TgVmCTjhbmc3nyCpheLAup4wlMupt_3MUwcw/) to finish register.
```
Grammar.
```suggestion
2. You **must** fill the [player signup form](https://docs.google.com/forms/u/0/d/e/1FAIpQLSfQGnO66J7b22TgVmCTjhbmc3nyCpheLAup4wlMupt_3MUwcw/) to register.
```

RushFTK replied:
Applied

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+#### Qualifiers
+
+1. After the screening phase has concluded, players eligible to play will participate in tournament play.
+2. In the first weekend, players will play 7 qualifiers maps in sequence up to 2 times, in a scheduled lobby (they may choose not to play the 2nd time if desired).
```
```suggestion
2. During the first weekend, players will play 7 Qualifiers in sequence up to 2 times in a scheduled lobby. Players may choose not to play the 2nd time if desired.
```

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+2. In the first weekend, players will play 7 qualifiers maps in sequence up to 2 times, in a scheduled lobby (they may choose not to play the 2nd time if desired).
+3. The 7 qualifiers maps will cover 4 sub-categories of LNs which are used to categorise all mappools after this stage:
+   - Stage 1: Early-game Releases
+   - Stage 2: Early to Mid-game Wildcard (Short Duration)
```
```suggestion
   - **Stage 2:** Early to mid-game wildcard (short duration)
```

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+   - Stage 5: Late-game Releases
+   - Stage 6: Mid to Late-game Wildcard (Long Duration)
+   - Stage 7: Speed Density
+4. This tournament will make use of the **Average Placement** method to seed players.
```
```suggestion
4. This tournament will make use of the **average placement** method to seed players.
```

Also, is there any link (either internal to the wiki or external to it) that could explain what the method is, or a formula for it if it's simple enough to add? Name should be self-explanatory, but rules should be as clear-cut as possible.

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+   - Stage 6: Mid to Late-game Wildcard (Long Duration)
+   - Stage 7: Speed Density
+4. This tournament will make use of the **Average Placement** method to seed players.
+5. In the qualification process, every player who signs up has to schedule for a lobby during the Qualfiers Stage, and play through all 7 qualifier maps in the mappool at least once, and up to 2 times.
```
```suggestion
5. During the qualification process, every player who signed up has to schedule for a lobby during the Qualfiers stage, and play through all 7 qualifier maps in the mappool at least once, and up to 2 times.
```

Additionally, this point is also redundant with point 2. I'd suggest either merging both, using one, or using the other. But keeping both seems unnecessary.

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+   - Stage 7: Speed Density
+4. This tournament will make use of the **Average Placement** method to seed players.
+5. In the qualification process, every player who signs up has to schedule for a lobby during the Qualfiers Stage, and play through all 7 qualifier maps in the mappool at least once, and up to 2 times.
+6. After completing the scheduled lobby, each player will have 1-2 scores recorded for each of the 7 qualifier maps. For each of the maps, the higher score of the two is taken and is used to provide the player's placement for each individual map.
```
Quite lengthy. Simplified + grammar.
```suggestion
6. After completing the scheduled lobby, each player will have up to 2 scores recorded for each of the 7 Qualifiers. For each map, the player's best placements will be used for seeding.
```

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+
+1. After both players have joined the lobby, both players will run the !roll command in the lobby chat.
+2. The player with the higher roll will pick first, and choose whether to protect first. If they choose to pick first, then they will ban second. The player with the lower roll receives the opposite of this.
+3. Following the ban/pick procedure, both players will alternate in picks, and play the picked map simultaneously. The player who obtains a higher score will win a point, and this process repeats until one of the players wins the required number of points to claim the win for the match.
```
```suggestion
3. Following the ban/pick phase, both players will alternate picks, and play the picked map simultaneously. The player who obtains the highest score will win a point. This process will repeat until one of the players wins the required number of points to win the match.
```

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+3. Following the ban/pick procedure, both players will alternate in picks, and play the picked map simultaneously. The player who obtains a higher score will win a point, and this process repeats until one of the players wins the required number of points to claim the win for the match.
+   - The required number of points to win the match varies for each stage.
+4. Each player receives the right to use a 2-minute timeout once per match. If players find the need to use the timeout between picks, they should raise their request to the referee facilitating their match.
+5. Players are encouraged to use their timeout only when an urgent situation or personal condition calls for it, and not to stall for time.
```
I'd say both of which are the same, as in the urgency of the "personal condition" calls for a timeout.
```suggestion
5. Players are encouraged to use their timeout only when an urgent situation calls for it, and not to stall for time.
```

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+
+1. Players are required to arrive for their match before the stipulated time for their match. Otherwise, a **10 minute** grace timer will be applied by the appointed referee for the match.
+2. If the player fails to arrive at their match before this timer runs out, the match will be considered as a win by default to the other player.
+3. If the player arrives at their match during this grace period, the time remaining on the timer will be brought forward to the ban/pick proceeding of the match.
```
Wording
```suggestion
3. If the player arrives to their match during this grace period, the time remaining on the timer will be carried over to the ban/pick phase of the match.
```

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+
+#### General conduct
+
+1. Every staff and player must respect each other and follow the osu! community rules, no discussion about sensitive topics (for example religion, politics), and anything that could be offensive to others.
```
I'd make the second part of this sentence (the "no sensitive topic discussions" clause) a whole separate point. These are two fairly different things (following rules versus no discussions).
```suggestion
1. Every staff and player must respect each other and follow the [osu! community rules](/wiki/Rules).
2. Discussions about sensitive (religion, politics) and other potentially offensive topics are disallowed.
```

Will need to re-index the bullet points bellow.

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+
+1. Every staff and player must respect each other and follow the osu! community rules, no discussion about sensitive topics (for example religion, politics), and anything that could be offensive to others.
+2. Keep the competition and atmosphere friendly amongst each other, whether it is in the match, in the Twitch chatroom, or on the Discord server. Tournaments are always for us to come together as a community and not engage in pointless arguments.
+3. In the event a player or staff member is found to have conducted themselves inappropriately towards anyone in the server, said individual can be subject to timeouts, warnings and bans from the tournament and its affiliated twitch channel based on the severity of the incident.
```
Is this only making reference to the Discord server, or the entire tournament? The point above seems to contradict that. Just want to clarify as right now this would technically only apply to the people in the server, and off-platform behaviour would not be punished; according to the current wording, that is.
```suggestion
3. In the event a player or staff member is found to have conducted themselves inappropriately towards anyone in the server, said individual can be subject to timeouts, warnings, and bans from the tournament and its affiliated Twitch channel based on the severity of the incident.
```

MChecaH reviewed (commented):
Second part of the review

RushFTK commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,387 @@
+
+#### Qualifiers
+
+1. After the screening phase has concluded, players eligible to play will participate in tournament play.
```
yes, but just make sure the context orgainzed in order, since some people only read parts of rule documents.

RushFTK commented:
I will waiting poly's review before make any changes, in case conflict happens.

polytetralhx committed:
Update en.md

Updated prize pools, staff list, and external links to YouTube VOD Channel
Slight grammar fixes

RushFTK committed:
Merge pull request #10 from polytetralhx/patch-2

Update en.md

RushFTK committed:
Applied parts of reviews with @MChecaH's version in o!mLN4 page

RushFTK committed:
Apply suggestions from code review

Applied Alll reviews from @MChecaH in o!mLN4 wiki page

Co-authored-by: MChecaH <93517213+MChecaH@users.noreply.github.com>

MChecaH commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,384 @@
+1. Every staff and player must respect each other and follow the [osu! community rules](/wiki/Rules).
+2. Discussions about sensitive (religion, politics) and other potentially offensive topics are disallowed.
+2. Keep the competition and atmosphere friendly amongst each other, whether it is in the match, in the Twitch chatroom, or on the Discord server. Tournaments are always for us to come together as a community and not engage in pointless arguments.
+3. In the event a player or staff member is found to have conducted themselves inappropriately towards anyone in the server, said individual can be subject to timeouts, warnings, and bans from the tournament and its affiliated Twitch channel based on the severity of the incident.
```
```suggestion
4. In the event a player or staff member is found to have conducted themselves inappropriately towards anyone in the server, said individual can be subject to timeouts, warnings, and bans from the tournament and its affiliated Twitch channel based on the severity of the incident.
```

MChecaH reviewed (commented):
Small miscellaneous suggestions after the latest update.

RushFTK commented on wiki/Tournaments/o!mLN/4/en.md:
```diff
@@ -0,0 +1,384 @@
+| Sheeter / Statistician | ::{ flag=US }:: [EpsilonMaiagare](https://osu.ppy.sh/users/3855052), ::{ flag=CN }:: [Yuiesta](https://osu.ppy.sh/users/13953619) |
+| Developer | ::{ flag=KR }:: [Nopekjk](https://osu.ppy.sh/users/4585186), ::{ flag=KR }:: [yhsphd](https://osu.ppy.sh/users/6665667) |
+| GFX Designer | ::{ flag=CN }:: [AlexDunk](https://osu.ppy.sh/users/9194799), ::{ flag=US }:: [Boolmaster Flex](https://osu.ppy.sh/users/5394681), ::{ flag=JP }:: [CrewK](https://osu.ppy.sh/users/11488604), ::{ flag=ID }:: [EndKun / Xion*](https://vgen.co/Xion2405), ::{ flag=MY }:: [Iyouka](https://osu.ppy.sh/users/7138499), ::{ flag=SE }:: [Melancholicc](https://osu.ppy.sh/users/18406374), ::{  ...
+| Composer | ::{ flag=CN }:: [Supa7onyz](https://osu.ppy.sh/u/5378805), ::{ flag=AU }:: [G4TE-16*](https://soundcloud.com/gate_16), ::{ flag=TH }:: [HowToPlayLN](https://osu.ppy.sh/users/10879600), ::{ flag=SG }:: [Hundotte*](https://www.youtube.com/c/hundotte), ::{ flag=US }:: [Ludicin](https://osu.ppy.sh/u/8659666), ::{ flag=US }:: [nm-y](https://osu.ppy.sh/u/31725557), ::{ flag=TH }:: [ohm002] ...
```
sorry i haven't check poly's edit so far thanks for double checking, i'm still reviewing last reviews one by one to see if they are successfully to applied, i will check later ^^

RushFTK committed:
Apply suggestions from code review

Co-authored-by: MChecaH <93517213+MChecaH@users.noreply.github.com>

Walavouchey reviewed (commented):
remaining stuff
- past tense everywhere except ruleset (because tournament is over)
- add badge images
- sentence case in staff table
- participants missing
- podium missing
- match results missing
- ci errors (`/u/` -> `/users/`)

RushFTK committed:
Update grand fianls mappools
